content/chainguard/chainguard-images/shared-responsibility-model/index.md

```diff
@@ -0,0 +1,167 @@
+6. SLSA Conformance: Built in accordance with [Supply-chain Levels for Software Artifacts](/open-source/slsa/what-is-slsa/) (SLSA) guidelines.
+7. Immutable and Verified: Signed and verified to ensure integrity throughout the software supply chain.
+8. SBOM Generation: Includes a build-time generated [Software Bill of Materials](/open-source/sbom/what-is-an-sbom/) for transparency and auditing.
+9. FIPS Compliance: Versions available that meet Federal Information Processing Standards.
+10. Rapid Patching: Covered by Chainguard's SLA for quick resolution of newly discovered vulnerabilities.
+
+### Use Cases
+* Ideal for organizations requiring secure, production-ready container images
+* Suitable for environments with stringent security and compliance requirements
+* Designed to integrate seamlessly into modern CI/CD pipelines and cloud-native architectures
+
+### App Images Shared Responsibilities
+* Apps Layer: Management of CVEs in the top-level application code.
+* Lang Libs Layer: Handling vulnerabilities in language-specific libraries built into the image (e.g., `go mod`, `pom`).
+
+Should a CVE be detected in any of these images we would likely bump a dependency to avoid in the next build. The outcome is two-fold: either bumping the dependency is successful or it breaks break the image. 
+
+* **Bumping a dependency is successful**: In this case, the upgrade does not cause the application to break,\ and the CVE is considered independently fixable. This falls under the "Release or Rebuild" condition for a qualifying patch in our SLAs, where the remediation involves rebuilding the image with updated compilers or libraries. The CVE clock starts, and Chainguard aims to resolve the vulnerability within the SLA's time frame.
+* **Bumping a dependency is breaking**: If the upgrade causes the application to break, the CVE is not considered independently fixable. Chainguard will file a "pending-upstream-fix" advisory, indicating that the vulnerability cannot be fixed without a fix from the upstream project. Chainguard cannot always service vulnerabilities from the "Lang Lib" (language libraries) layer, unfortunately. The CVE clock does not start in this case, as the vulnerability is not considered fixable under the SLA.
+
+> **Note**: To learn more about Chainguard's Security Advisories, please refer to our [overview on the subject](/chainguard/chainguard-images/working-with-images/security-advisories/). We also encourage you to check out the [Security Advisories page](https://images.chainguard.dev/security) directly. 
+
+### Chainguard's Responsibilities in App Images
+* Toolchains Layer: CVE removal in development tools and runtimes (e.g., `go`, `java`, `php`).
+* System Layer: Addressing vulnerabilities in base system libraries and utilities (e.g., `glibc`, `openssl`).
+
+### Customer's Responsibilities in App Images
+* Any added libraries or APK packages are the customer's responsibility.
```

Comment:
Just to be clear here, we are talking about wolfi packages here? Or apks that the customer has created?

Reply:
I'm not sure, @MordodeMaru do you have insight?